src/app/checkout/step-3/step-3.tpl.html

```diff
@@ -213,16 +212,6 @@
                            button-classes="'btn btn-primary btn-lg btn-block'"
                            button-disabled="!$ctrl.canSubmitOrder()"
                            button-label="'SUBMIT_GIFT'"></recaptcha-wrapper>
-                  <button
```

Comment:
I assume branded checkout is now using recaptcha as well?

Reply:
It will after this PR, yes.